docs/axisman.rst

```diff
@@ -72,14 +72,22 @@ hold other AxisManagers, and some other stuff too).
 
     dset = core.AxisManager().wrap('tod', tod)
 
+AxisManagers can also hold scalars (ints, floats, strings, etc.), note that ``None`` is considered a scalar as well.
```

Comment:
For docs, please wrap lines at 78 chars or so (like the rest of the file).

What is included in "etc" in this line?  It's probably worth mentioning a scalar is anything that np.isscalar thinks is a scalar.